### PR TITLE
NEXTEUROPA-12809: Notice when producing content with taxonomy term reference fields

### PR DESCRIPTION
**Pull request by ademarco**

### CODE REVIEW

File: tests/features/integration/integration_producer.feature [hunk 1]
```diff
@@ -1,37 +1,119 @@
...
+    """
     When I am viewing a multilingual "page" content:
       | language | title               |
       | pt-pt    | Título em Português |
     And I run drush "integration-export http_mock test_news"
     Then the central Integration server received content in the following languages:
       | language | title               |
       | pt       | Título em Português |
+
+  Scenario: producers will produce a valid document for content containing title, body and taxonomy term reference.
```
Comment (cyberwolf) [inline tests/features/integration/integration_producer.feature]: This scenario misses a "When".

File: tests/features/integration/integration_producer.feature [hunk 2]
```diff
@@ -1,37 +1,119 @@
...
+    """
+    And the "article" producer builds the following document for the "article" with title "Article title 2":
+    """
+      version: v1
+      default_language: und
+      type: article
+      languages:
+        - und
+      fields:
+      fields:
```
Comment (cyberwolf) [inline tests/features/integration/integration_producer.feature]: One occurrence of 'fields:' can be removed.

File: tests/features/integration/integration_producer.feature [hunk 3]
```diff
@@ -1,37 +1,119 @@
...
       | pt       | Título em Português |
+
+  Scenario: producers will produce a valid document for content containing title, body and taxonomy term reference.
+    Given "tags" terms:
+      | name  |
+      | Tag 1 |
+      | Tag 2 |
+    And "article" content:
+      | title           | body          | field_tags   |
+      | Article title 1 | Article body 1 | Tag 1, Tag 2 |
```
Comment (cyberwolf) [inline tests/features/integration/integration_producer.feature]: The column header "field_tags" should be indented one space more.

**Comment by cyberwolf** [comment]
👍

**Comment by jfhovinne** [comment]
:+1:

**Pull request merged**